src/app/pages/transaction-request/components/transaction-error/error-messages.tsx

```diff
@@ -116,44 +118,11 @@ export const IncorrectContractAddressMessage = memo(props => {
   );
 });
 
-export const UnauthorizedErrorMessage = memo(props => {
-  useScrollLock(true);
-  return (
-    <Fade in>
-      {styles => (
-        <Flex
-          position="absolute"
-          width="100%"
-          height="100vh"
-          zIndex={99}
-          left={0}
-          top={0}
-          alignItems="center"
-          justifyContent="center"
-          p="loose"
-          bg="rgba(0,0,0,0.35)"
-          backdropFilter="blur(10px)"
-          style={styles}
-        >
-          <ErrorMessage
-            title="Unauthorized request"
-            body="The transaction request was not properly authorized by any of your accounts. If you've logged in to this app before, then you might need to re-authenticate into this application before attempting to sign a transaction with the Hiro Wallet."
-            border={'1px solid'}
-            borderColor={color('border')}
-            boxShadow="high"
-            css={{
-              '& > *': {
-                pointerEvents: 'all',
-              },
-            }}
-            {...props}
-          />
-        </Flex>
-      )}
-    </Fade>
-  );
+export const UnauthorizedRequestErrorMessage = memo(() => {
```

Comment:
Maybe rename this to be clear that it redirects, I can imagine "Why is this redirect coming from?" being hard to find.

Reply:
Ok, will do, thx.